doc/radosgw/encryption.rst

```diff
@@ -43,6 +43,102 @@ integration with `Barbican`_, `Vault`_, and `KMIP`_ are implemented.
 See `OpenStack Barbican Integration`_, `HashiCorp Vault Integration`_,
 and `KMIP Integration`_.
 
+Bucket Encryption Subresource
+=============================
+
+Amazon S3 supports APIs to manage the encryption subresource to
+configure default encryption. Default encryption for a bucket can use
+server-side encryption with Amazon S3-managed keys (SSE-S3) or AWS KMS customer
```

Comment:
with Ceph managed server-side encryption keys (SSE-S3). We only support SSE-S3 keys in BucketEncryption APIs. Support for SSE-KMS based keys in BucketEncryption APIs is not available.